templates/topic.html

```diff
@@ -42,12 +49,14 @@ <h2>{{ .Topic.Title }}</h2>
-            </a>
-            <a href="{{ $.BasePath }}categories/{{ $.Category.CategoryID }}/topics/{{ $.Topic.TopicID }}/comments/new?quote=topic" title="Quote comment">
-                <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="16" height="16" viewBox="0 0 172 172" style=" fill:#000000;"><g fill="none" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><path d="M0,172v-172h172v172z" fill="none"></path><g fill="#666666"><path d="M86,13.76c-43.59125,0 -79.12,30.54344 -79.12,68.78656c0,22.18531 12.1475,41.50844 30.82563,54.12625c-0.02688,0.73906 0.02687,1.94844 -0.94063,5.54969c-1.20937,4.47469 -3.655,10.76344 -8.6,17.85844l-3.52062,5.03906h6.15437c21.285,0 33.60719,-13.88094 35.51531,-16.11156c6.3425,1.49156 12.85969,2.29781 19.68594,2.29781c43.57781,0 79.12,-30.54344 79.12,-68.77312c0,-38.22969 -35.52875,-68.77313 -79.12,-68.77313zM86,20.64c39.82875,0 72.24,27.76188 72.24,61.90656c0,34.13125 -32.41125,61.89312 -72.24,61.89312c-6.08719,0 -12.01312,-0.69875 -18.11375,-2.12312l-4.085,-0.95406l-2.71437,3.17125c-2.35156,2.75469 -10.26625,10.84406 -23.30063,13.115c2.49938,-4.48812 4.38063,-9.05687 5.60344,-13.63906c0.9675,-3.5475 1.14219,-5.41531 1.1825,-7.05469l0.16125,-3.82969l-3.18469,-2.15c-17.65687,-11.94594 -27.78875,-29.60281 -27.78875,-48.42875c0,-34.14469 32.41125,-61.90656 72.24,-61.90656zM58.48,58.48v24.08h10.32c0,2.27094 -0.60469,12.04 -10.32,17.69719v11.38156c15.09031,-5.805 20.64,-18.82594 20.64,-29.07875v-24.08zM92.88,58.48v24.08h10.32c0,2.27094 -0.60469,12.04 -10.32,17.69719v11.38156c15.09031,-5.805 20.64,-18.82594 20.64,-29.07875v-24.08z"></path></g></g></svg>
-            </a>
+            {{ if eq .Topic.IsClosed false}}
+                <a href="{{ $.BasePath }}categories/{{ $.Category.CategoryID }}/topics/{{ $.Topic.TopicID }}" title="Link to topic">
```

Comment:
The link and edit comment can be allowed. Only quote comment needs to be hidden

Reply:
I think you're still hiding the link to topic when it is readonly. We need to hide only quote comment. I think you got it right for the comments but not the main comment associated with the topic itself.